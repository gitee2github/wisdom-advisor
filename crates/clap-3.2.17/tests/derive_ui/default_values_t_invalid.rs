use clap::Parser;

#[derive(Parser, Debug)]
#[clap(name = "basic")]
struct Opt {
    #[clap(default_values_t = [1, 2, 3])]
    value: u32,
}

fn main() {
    let opt = Opt::parse();
    println!("{:?}", opt);
}
