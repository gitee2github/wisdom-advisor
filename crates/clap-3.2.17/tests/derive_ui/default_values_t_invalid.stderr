error: #[clap(default_values_t)] can be used only on Vec types

  = note: see https://github.com/clap-rs/clap/blob/master/examples/derive_ref/README.md#magic-attributes

 --> tests/derive_ui/default_values_t_invalid.rs:6:12
  |
6 |     #[clap(default_values_t = [1, 2, 3])]
  |            ^^^^^^^^^^^^^^^^
